community/php7/APKBUILD

```diff
@@ -535,6 +539,14 @@ phar() {
 	fi
 }
 
+default_exts() {
+	pkgdesc="PHP$_suffix metapackage for all Core Extensions and Bundled Extensions"
+	depends=$(printf '%s\n' $_extensions \
+		| sed -En "s/(\w+)\*/$pkgname-\1/p")
+
+	mkdir -p "$subpkgdir"
+}
```

Comment:
I think it's pretty good.
But all those plussed ones are 'bundled' not 'core'
(http://php.net/manual/en/extensions.membership.php)
So IMHO this subpackage need to be named php7-bundled
and description like "PHP$_suffix metapackage for all Bundled Extensions"